test/image/mocks/gl2d_12.json

```diff
@@ -734,7 +734,7 @@
                 "size": 0,
                 "color": ""
             },
-            "exponentformat": "e",
```

Comment:
@etpinard I suppose since this behavior has existed so long, this could be considered a breaking change... but it also seems to me to be clearly wrong. Are you OK with calling it a bug?

Reply:
Yes, I'm ok calling this a bug as `"e"` isn't the `exponentformat` default, so I can't happen "by accident".